sequencer/src/main/java/net/consensys/linea/sequencer/txselection/LineaTransactionSelectorPlugin.java

```diff
@@ -81,9 +82,15 @@ public void start() {
     super.start();
     final LineaTransactionSelectorConfiguration txSelectorConfiguration =
         transactionSelectorConfiguration();
+    final LineaRejectedTxReportingConfiguration lineaRejectedTxReportingConfiguration =
+        rejectedTxReportingConfiguration();
     rejectedTxJsonRpcManager =
-        Optional.ofNullable(txSelectorConfiguration.rejectedTxEndpoint())
-            .map(endpoint -> new JsonRpcManager(besuConfiguration.getDataPath(), endpoint).start());
+        Optional.ofNullable(lineaRejectedTxReportingConfiguration.rejectedTxEndpoint())
+            .map(
+                endpoint ->
+                    new JsonRpcManager(
+                            besuConfiguration.getDataPath(), lineaRejectedTxReportingConfiguration)
+                        .start());
```

Comment:
Introduced a "pluginIdentifier" that will be used as a sub-directory under the `"besuDataDir/rej-tx-rpc/pluginIdentifier"`. This will allow multiple instances to be created from different plugins simultaneously.